samples/places-autocomplete-addressform/src/index.njk

```diff
@@ -51,6 +51,7 @@ https://developers.google.com/maps/documentation/javascript/examples/places-auto
     <input id="country" name="country" required>
   </label>
   <button type="button" class="my-button">Save address</button>
+  <input type="reset" value="Clear form">
```

Comment:
why not make this a button?

Reply:
Input type reset is a button :)

Reply:
![](https://i.kym-cdn.com/photos/images/newsfeed/000/620/434/f9e.gif)